code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm

```diff
@@ -92,24 +92,24 @@
 	var/turf/target_turf = xeno.loc
 
 	if(!istype(target_turf))
-		to_chat(xeno, SPAN_WARNING("You can't do that here."))
+		to_chat(xeno, SPAN_WARNING("You cannot plant a fruit without a weed garden."))
 		return
 
 	var/obj/effect/alien/weeds/target_weeds = locate(/obj/effect/alien/weeds) in target_turf
 	if(!target_weeds)
-		to_chat(xeno, SPAN_WARNING("There's no weed to place it on!"))
+		to_chat(xeno, SPAN_WARNING("The are no weeds to plant a fruit within!"))
 		return
 
 	if(target_weeds.hivenumber != xeno.hivenumber)
-		to_chat(xeno, SPAN_WARNING("This weed is toxic to the fruit. Can't plant it here!"))
+		to_chat(xeno, SPAN_WARNING("These weeds do not belong to your hive; they reject your fruit."))
 		return
 
 	if(locate(/obj/effect/alien/resin/trap) in range(1, target_turf))
-		to_chat(xeno, SPAN_XENOWARNING("This is too close to a resin hole!"))
+		to_chat(xeno, SPAN_XENOWARNING("This location is too close to a resin hole!"))
 		return
 
 	if(locate(/obj/effect/alien/resin/fruit) in range(1, target_turf))
-		to_chat(xeno, SPAN_XENOWARNING("This is too close to another fruit!"))
+		to_chat(xeno, SPAN_XENOWARNING("This location too close to another fruit!"))
```

Comment:
location is

Reply:
> location is

I fixed it before I had to do a soft reset to squash everything so it was reverted. Sorry!

Reply:
no prob